src/main/java/com/everis/cars/exceptions/CarNotFoundExceptionMapper.java

```diff
@@ -5,14 +5,34 @@
 import javax.ws.rs.ext.ExceptionMapper;
 import javax.ws.rs.ext.Provider;
 
+import org.apache.log4j.Logger;
+
 import com.everis.cars.entity.ErrorMessage;
 import com.everis.cars.entity.ErrorMessageCollection;
 
+/**
+ * ExceptionMapper to catch CarNotFound exceptions and standardize exception responses
+ */
 @Provider
 public class CarNotFoundExceptionMapper implements ExceptionMapper<CarNotFoundException> {
+	
+	/**
+	 * Logger instance
+	 * 
+	 * @see org.apache.log4j.Logger
+	 */
+	private static Logger logger = Logger.getLogger(CarNotFoundException.class);
 
+	/**
+	 * toResponse override to implement the custom response message 
+	 * 
```

Comment:
purpose of this? 404 in resources when...

Reply:
I explained better this methods in exception mappers.